generated/react-native-flexible-sync/src/TaskSchema.js

```diff
@@ -1,5 +1,4 @@
 import {BSON} from 'realm';
```

Comment:
I think you have both a 'TaskSchema.js' and a 'ItemSchema.js' file in both versions of the generated folders. Can you fix this?